content/applications/hr/employees/new_employee.rst

```diff
@@ -100,82 +82,142 @@ entry and create another line.
-- :guilabel:`Progress`: Progress is automatically selected based on the selected :guilabel:`Skill
-  Level`. Skill levels and progress can be modified in the :guilabel:`Skill Type` pop-up form, which
-  is accessed via the :guilabel:`External Link` button next to :guilabel:`Skill Type` field.
+  Type` appear in a drop-down menu. Select a skill level, then the progress bar automatically
+  displays the pre-defined progress for that skill level. Skill levels and progress can be modified
+  in the :guilabel:`Skill Level` pop-up form, which is accessed via the :guilabel:`External Link`
+  button next to :guilabel:`Skill Level` field.
 
-To delete any line from the :guilabel:`Resumé` tab, click the :guilabel:`🗑️ (trash can)` delete icon
-to delete the entry. Add a new line by clicking the :guilabel:`ADD` button next to the corresponding
+Once all the information is entered, click the :guilabel:`Save & Close` button if there is only one
+entry to add, or click the :guilabel:`Save & New` button to save the current entry and create
+another skill.
+
+To delete any line from the :guilabel:`Resumé` tab, click the :guilabel:`🗑️ (trash can)` icon to
+delete the entry. Add a new line by clicking the :guilabel:`Add` button next to the corresponding
 section.
 
-Work information tab
+.. _employees/skill-types:
+
+Skill Types
+***********
+
+In order to add a skill to an employee's form, the :guilabel:`Skill Types` must be configured. Go to
+:menuselection:`Employees app --> Configuration --> Skill Types` to view the currently configured
+skill types and create new skill types. Click :guilabel:`Create` and a new :guilabel:`Skill Type`
+form appears. Fill out all the details and then click :guilabel:`Save`. Repeat this for all the
+skill types needed.
+
+- :guilabel:`Skill Type`: Enter the name of the skill type. This should be somewhat generic, since
+  the specific skills listed will be housed under this category.
+- :guilabel:`Skills`: Click :guilabel:`Add a line` and enter the information for the new skill, then
+  repeat for all other needed skills.
+- :guilabel:`Levels`:  Click :guilabel:`Add a line` and a :guilabel:`Create Levels` form appears.
+  Enter the name of the level, and assign a percentage (0-100) for that level. Click :guilabel:`Save
+  & New` to save the entry and add another level, or click :guilabel:`Save & Close` to save the
+  level and close the form.
+
+  .. example::
+     To add a math skill set, enter `Math` in the :guilabel:`Name` field. Next, in the
+     :guilabel:`Skills` field, enter `Algebra`, `Calculus`, and `Trigonometry`. Last, in the
+     :guilabel:`Levels` field enter `Beginner`, `Intermediate`, and `Expert`, with the
+     :guilabel:`Progress` listed as `25`, `50`, and `100`, respectively. Then, either click
+     :guilabel:`Save & Close` or :guilabel:`Save & New`.
+
+       .. image:: new_employee/math-skills.png
+          :align: center
+          :alt: Add new math skills and levels with the skill types form.
```

Comment:
Image alignment, need to scoot back a few spaces to be inline with the `e` in `.. example::`
```suggestion
     .. image:: new_employee/math-skills.png
        :align: center
        :alt: Add new math skills and levels with the skill types form.
```